### ISSUE TITLE
<layer->.focus() is misnamed, should be split up and renamed

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#587]
There is a standard [HTMLElement.focus() method](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/focus), which doesn't do what this method does, which is to zoom to the whole layer at the maximum possible zoom.  We will need, at a minimum, to rename this method.

Since programmatic focus has become important, the HTMLLayerElement.focus() implementation should either focus the layer in the layer control, or focus the first or next element in the layer, without moving the map.  We should figure out what the correct behaviour is, but I'm pretty sure the correct behaviour is not what it currently does.  We probably need a separate method for each type of focus, actually.

**Comment by Malvoz** [comment]
> the HTMLLayerElement.focus() implementation should either focus the layer in the layer control, or focus the first or next element in the layer, without moving the map. We should figure out what the correct behaviour is

In any case, I think it should _not_ be the layer control since a "layer" refers to the container (of features etc.) on the map itself, separate from the layer control?

Another option to consider is to enable programmatic focus (`tabindex="-1"`) on the `<div class="mapml-layer ...">` (this would make even more sense if each of those layer elements also had an accessible name: https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/579). One use case for that can be found in https://maps4html.org/experiments/screenreader/closest-restaurants/: after the "Restaurants" button has been pressed AT/keyboard users could/should be directed (have focus delegated) to the relevant features?

**Comment by prushforth** [comment]
We are leaning towards renaming the behaviour of the current `layer-.focus()` method to `layer-.zoomTo()`.
